pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -421,6 +422,43 @@ func (r *reconcilerBase) deployment(ctx context.Context, dRef client.ObjectKey)
 	return deployObj, nil
 }
 
+func mountConfigMapValuesFiles(templateSpec *corev1.PodSpec, c *corev1.Container, valuesFrom []v1beta1.ValuesFrom) {
```

Comment:
Can you add some unit tests which verify the shape of the resulting spec?

Reply:
done